### PR TITLE
LPS-86724 message-boards-service: Integrate MBThread and MBMessage into new indexer model; Deprecate MBThread and MBMessage legacy indexer.

### PR DESCRIPTION
**Pull request by luanmaoski**

### CODE REVIEW

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by luanmaoski** [comment]
ci:test:sf

**Comment by luanmaoski** [comment]
ci:test:search

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/search/spi/model/index/contributor/MBMessageModelIndexerWriterContributor.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+					WorkflowConstants.STATUS_IN_TRASH
+				};
+
+				dynamicQuery.add(statusProperty.in(statuses));
+
+			});
+
+		batchIndexingActionable.setPerformActionMethod(
+			(MBMessage mbMessage) -> {
+				if (_log.isDebugEnabled()) {
```
Comment (luanmaoski) [inline modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/search/spi/model/index/contributor/MBMessageModelIndexerWriterContributor.java]: Hi @brandizzi  , the three `MB` tests are also failing before [migration](https://github.com/luanmaoski/liferay-portal/pull/19).

For some reason the `_log` is disable. Also, when i run the source format locally it get stuck at 69%.

**Comment by luanmaoski** [comment]
ci:test:sf

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Pull request submitted to https://github.com/brandizzi/liferay-portal/pull/625. [See changes here.](https://github.com/brandizzi/liferay-portal/compare/91b9a3cc3d3a~2...91b9a3cc3d3a) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**